### PR TITLE
Use cats-effect IO data type

### PR DESCRIPTION
**Pull request by calvellido**
* Adds `cats-effect` as a dependency.
* Updates `sbt-org-policies` to `0.13.2` properly.
* Uses IO data type when committing a directory.

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -17,9 +17,10 @@ object ProjectPlugin extends AutoPlugin {
   object autoImport {
 
     lazy val V = new {
+      val catsEffect: String   = "1.1.0"
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala]: Could we use the latest one? 2.x

Comment (juanpedromoreno) [inline project/ProjectPlugin.scala]:
```suggestion
      val catsEffect: String   = "2.1.2"
```

**Comment by calvellido** [comment]
Gracias @franciscodr :+1:

File: project/ProjectPlugin.scala (hunk 1 above)
Comment (calvellido) [reply project/ProjectPlugin.scala]: Sure!

**Comment by calvellido** [comment]
Gonna merge after checking it's working the same with the version bump 🤓

**Pull request merged**